sdk/core/Azure.Core.Experimental/src/Azure.Core.Experimental.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="Microsoft.CSharp" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
This is most likely needed in the short-term due to the package bump.   Please add a comment here documenting that and open an issue to remove this once the next version of Azure.Core is released.

Reply:
Created https://github.com/Azure/azure-sdk-for-net/issues/44165 for tracking